### DISCUSSION TITLE
Calculate the standard deviation for transition probabilities matrix

### DISCUSSION THREAD

**Discussion by NikosMemmos** [bmcclintock/momentuHMM#187] [Q&A] [answered]
I would be interested in knowing the standard deviation of the transition probability matrix. For one of my models, I get a small transition probability and I would like to test if it is significant or it can be assumed it is zero.

Thank you!

**Comment by bmcclintock** [comment, marked as answer]
The standard errors of the working-scale and real-scale parameter estimates can be accessed through the `CIbeta` and `CIreal` model output. Using the example `model` provided with the package:
```
library(momentuHMM)
model <- example$m
model$CIbeta$beta # working scale
model$CIreal$gamma # real scale
```
